wallet/invoices.c

```diff
@@ -115,6 +115,12 @@ static void wallet_stmt2invoice_details(const tal_t *ctx,
 	}
 
 	dtl->bolt11 = tal_strndup(ctx, sqlite3_column_blob(stmt, 9), sqlite3_column_bytes(stmt, 9));
+	if (sqlite3_column_type(stmt, 10) != SQLITE_NULL)
+		dtl->description =
+		    tal_strndup(ctx, sqlite3_column_blob(stmt, 10),
+				sqlite3_column_bytes(stmt, 10));
```

Comment:
`tal_strdup(ctx, sqlite3_column_text(stmt, 10))`

Side note: Really, this function and its callers are weird.  They should allocate and return dtl, and allocate all members with that as parent.

Reply:
Agreed, the function is rather strange, but I kept it as is to keep changes minimal. Might be a good time to fix this up though.